modules/create.js

```diff
@@ -0,0 +1,15 @@
+import { render, createBooks } from './app.js';
+
+const addTitle = document.querySelector('.add-title');
+const addAuthor = document.querySelector('.add-author');
+
+const addBooks = () => {
+  const title = addTitle.value;
+  const author = addAuthor.value;
+
+  createBooks(title, author);
+  render();
+};
```

Comment:
- [OPTIONAL] After adding a book, it would be nice to add a message that informs users that a book was added. This would improve the User experience. 👍